app/assets/javascripts/arctic_admin/main.js

```diff
@@ -73,8 +72,8 @@ $(function () {
   })
 
   $(document).on('click touchstart', 'body', function (e) {
-    var tabs = $('#tabs')
-    var width = Math.round(tabs[0].getBoundingClientRect().width)
+    var tabs = $('#tabs')    
+    var width = (tabs.length==0)? 0 : Math.round(tabs[0].getBoundingClientRect().width)    
```

Comment:
```suggestion
    var tabs = $('#tabs')
    var width = (tabs.length==0)? 0 : Math.round(tabs[0].getBoundingClientRect().width)
```

juste for remove useless spaces